notebooks/ArynPartitionerExample.ipynb

```diff
@@ -27,18 +30,29 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "aryn_api_key = 'YOUR-ARYN-API-KEY'"
+    "from sycamore.utils.aryn_config import ArynConfig, DEFAULT_PATH\n",
```

Comment:
If someone gets this version of the notebook, and the older (in pypi) version of the library it won't work.
The safer way to do this is to import _DEFAULT_PATH here, and then once we release the library with
DEFAULT_PATH switched to change the notebooks.

Reply:
Done.